docs/workWithSpreadsheet.md

```diff
@@ -0,0 +1,220 @@
+# Working with Spreadsheet Operation
+
+[ [Overview](#overview) ]  [ [Operation details](#operation-details) ]  [ [Sample configuration](#sample-configuration) ]
+
+## Overview
+
+The following operations allow you to carry out Spreadsheet operations. Click an operation name to see details on how to use it.
+
+
+| Operation | Description |
+| ------------- |-------------|
+| [createSpreadsheet](#Create-Spreadsheet) | Creates a new spreadsheet |
+
+## Operation details
+
+This section provides details on each of the operations.
+
+#### Create Spreadsheet
+
+This method allows you to creates a new spreadsheet by optionally add sheets to spreadsheet, set the spreadsheet id and sheet properties, and add named ranges.
```

Comment:
This method allows you to create a new spreadsheet by setting the spreadsheet ID, sheet properties, and add named ranges.